tests/src/Common/test_runtime/project.json

```diff
@@ -2,9 +2,6 @@
   "dependencies": {
     "Microsoft.NETCore.Platforms": "2.0.0-preview1-25210-01",
     "Microsoft.DotNet.CoreCLR.TestDependencies": "1.0.0-prerelease",
-    "jit-dasm": "0.0.1.4",
-    "cijobs": "0.0.1.2",
-    "jit-analyze": "0.0.1.1"
```

Comment:
i don't think we have separate sources. But adding a nuget.config to the folder will do the trick.
But i would prefer it to be global though 

 ```<add key="corefxlab" value="https://dotnet.myget.org/F/dotnet-corefxlab/api/v3/index.json" />```

should do it for the short term. @weshaggard any suggestions for the long term solution